theme_anelusia/views/snippets/s_features_wall.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wall" inherit_id="website.s_features_wall">
+    <!-- Section -->
+    <xpath expr="//section" position="attributes">
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/09"}</attribute>
+    </xpath>
+    <!-- Shape -->
+    <xpath expr="//div[hasclass('o_we_shape')]" position="replace">
+        <div class="o_we_shape o_web_editor_Floats_09 o_we_animated"/>
+    </xpath>
+    <!-- Card 1 -->
+    <xpath expr="//div[hasclass('g-col-lg-3')]//h5" position="replace" mode="inner">
```

Comment:
If you replace `<h5>` by `<h3>`, don't forget to update the xpath (other occurrences as well).